specs/commandline/key.md

```diff
@@ -115,10 +115,22 @@ notation key list
 
 Upon successful execution, a list of keys is printed out with information of name, key path, certificate path, key id and plugin name. The default signing key name is preceded by an asterisk. The key id and plugin name are used together to provide the information of the key identifier for the remote key and the plugin associated with it.
 
-### Delete two keys from signing key list
+### Remove a specified key from signing key list
```

Comment:
Is `specific` better?

Reply:
No. I think `specified` points to a particular thing here. See https://www.askdifference.com/specific-vs-specified/